libcudacxx/test/libcudacxx/std/numerics/numbers/illformed.fail.cpp

```diff
@@ -0,0 +1,30 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+// SPDX-FileCopyrightText: Copyright (c) 2025 NVIDIA CORPORATION & AFFILIATES.
+//
+//===----------------------------------------------------------------------===//
+
+// <cuda/std/numbers>
+
+// UNSUPPORTED: c++11
+
+#include <cuda/std/numbers>
+
+// Initializing the primary template is ill-formed.
+int log2e{cuda::std::numbers::log2e_v<int>}; // expected-error-re@numbers:* {{static assertion failed{{.*}}A program
+                                             // that instantiates a primary template of a mathematical constant variable
+                                             // template is ill-formed.}}
```

Comment:
Note: We currently do not implement those fail tests, but the message must exactly match the static assert